autoload/coc/powershell.vim

```diff
@@ -1,18 +1,34 @@
 let s:root           = expand('<sfile>:h:h:h')
 let s:is_win         = has('win32') || has('win64')
-let s:is_mac = !s:is_win && !has('win32unix')
+let s:is_mac         = !s:is_win && !has('win32unix')
       \ && (has('mac') || has('macunix') || has('gui_macvim') ||
       \   (!isdirectory('/proc') && executable('sw_vers')))
 let s:is_vim         = !has('nvim')
-let s:install_script = s:root.'/'.(s:is_win ? 'install.cmd' : 'install.ps1')
+let s:install_script = s:root.'/install.ps1'
 
-if(s:is_mac)
-    let s:install_script = 'pwsh '.s:install_script
-endif
+function! coc#powershell#install(...)
+    if(exists('a:1.flags'))
```

Comment:
I like this! I think my one thought is... Since the install script only has one parameter, it might be nice to just make that scenario streamline.

A config could look like:

```vim
{
    powershellExecutable: "pwsh"
    preview: true
}
```
Then we check if preview and add AllowPrerelease

Reply:
My reasoning for having it be flags and just appending it was if we enhance the script in the future we don't have to change the function, although that being said, from a user perspective it's probably easier to do it this way.

Unfortunately: unless there's a way I'm not seeing, vimscript doesn't really understand True vs False as values. It uses the "0 is false, all other integers are true" method, so it would need to be:

```vim
{
    preview: 1
}
```

But that's still easier to explain. I have played around with using `"True"` and `"true"` but that doesn't evaluate to true it seems.

Reply:
Isn't there like... `v:true` or `v:True` variable?

Reply:
Yes, v:true works 😄